network/ServerNetworking.cpp

```diff
@@ -485,7 +485,7 @@ void ServerNetworking::SetHostPlayerID(int host_player_id)
 { m_host_player_id = host_player_id; }
 
 void ServerNetworking::Init() {
-    tcp::endpoint endpoint(tcp::v4(), Networking::MessagePort());
+    tcp::endpoint endpoint(tcp::v6(), Networking::MessagePort());
```

Comment:
What.
It means "dual stack". It used to mean "ipv6 only" long time ago i guess but nobody cares enough to rename it properly in the socket interface headers.